resip/stack/test/testStack.cxx

```diff
@@ -603,7 +603,7 @@ main(int argc, char* argv[])
    const char* logLevel = "WARNING";
    const char* proto = "tcp";
    const char* bindAddr = "127.0.0.1";
-   const char* domain = "";
+   const char* domain = "resiprocate.test";
```

Comment:
According to [RFC2602](https://www.rfc-editor.org/rfc/rfc2606.html#page-2) the `test` TLD is reserved.